test/asImageWidthSrcSet.test.ts

```diff
@@ -0,0 +1,106 @@
+import { ImageField } from "@prismicio/types";
+import test from "ava";
+
+import { asImageWidthSrcSet } from "../src";
+
+test("returns an image field src and width-based srcset", (t) => {
+	const field: ImageField = {
+		url: "https://images.prismic.io/qwerty/image.png?auto=compress%2Cformat",
+		alt: null,
+		copyright: null,
+		dimensions: { width: 400, height: 300 },
+	};
+
+	t.deepEqual(
+		asImageWidthSrcSet(field, {
+			widths: [400, 800, 1600],
```

Comment:
Same as with `asImagePixelDensitySrcSet()`, maybe don't use the default here :)

Reply:
Updated :)